src/test/java/gmailgdogra/SwipeProcessorServiceTest.java

```diff
@@ -165,21 +191,43 @@ private MockMultipartFile getFile(String pathname) throws IOException {
     }
 
     @Test
-    void testFile1() throws IOException {
+    void testInOutSwipesPrevTwoShiftsForFile1() throws IOException {
         MockMultipartFile testFile1 = getFile("src/main/resources/test file 1.xlsx");
         List<SwipeRecord> allSwipes = readXlsxService.readAllRows(testFile1.getInputStream());
         List<Shift> shifts = getShiftsForTestFile1();
-        List<OutputRow> results = swipeProcessorService.getOutputDataFrom(allSwipes, shifts);
-        assertTrue(results.containsAll(expectedResultsTestFile1));
+        swipeProcessorService.prepareData(allSwipes, shifts);
+        List<OutputRow> results = swipeProcessorService.getInOutSwipesPrevTwoShifts();
+        assertTrue(results.containsAll(inOutSwipesPrevTwoShiftsFromFile1), "Daily report not as expected");
+    }
+
+    @Test
+    void testInOutSwipesPrevTwoShiftsForFile2() throws IOException {
+        MockMultipartFile testFile2 = getFile("src/main/resources/test file 2.xlsx");
+        List<SwipeRecord> allSwipes = readXlsxService.readAllRows(testFile2.getInputStream());
+        List<Shift> shifts = getShiftsForTestFile2();
+        swipeProcessorService.prepareData(allSwipes, shifts);
+        List<OutputRow> results = swipeProcessorService.getInOutSwipesPrevTwoShifts();
+        assertTrue(results.containsAll(inOutSwipesPrevTwoShiftsFromFile2), "Daily report not as expected");
+    }
+
+    @Test
+    void testCurrentShiftSwipeInsForFile1() throws IOException {
+        MockMultipartFile testFile2 = getFile("src/main/resources/test file 1.xlsx");
+        List<SwipeRecord> allSwipes = readXlsxService.readAllRows(testFile2.getInputStream());
+        List<Shift> shifts = getShiftsForTestFile1();
+        swipeProcessorService.prepareData(allSwipes, shifts);
+        List<OutputRow> results = swipeProcessorService.getInSwipesCurrentShift();
+        assertTrue(results.containsAll(inSwipesCurrentShiftFromFile1), "Current Shift Swipe-in not as expected");
```

Comment:
better to insert an empty line between the preparation and the assertion parts, to make it easier to identify visually